cmake_modules/commonSetup.cmake

```diff
@@ -739,4 +739,16 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
         endif()
   ENDMACRO()
 
+  ###########################################################################
+  ###
+  ## The following sets the dependency list for a package
+  ###
+  MACRO(SET_DEPENDENCIES dependencies cpackvar)
```

Comment:
certainly.  That's a good idea too, that way it's more consistent with the way the built in set function works.